src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -67,16 +67,6 @@ protected function tearDown()
     }
 
     /**
-     * @expectedException \PHPUnit_Framework_Error
-     */
-    public function testChoicesOptionExpectsArray()
```

Comment:
This test will be totally different in 2.7 anyway with the changes to Choice type.